prow/jobs/rafter/rafter.yaml

```
@@ -0,0 +1,88 @@
+job_template: &job_template
+  skip_report: false
+  decorate: true
+  optional: false
+  path_alias: github.com/kyma-project/rafter
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/buildpack-golang-kubebuilder:v20190208-813daef
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/rafter"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-kyma: "true"
+
+presubmits: # runs on PRs
+  kyma-project/rafter:
+    - name: pre-master-rafter
```

```suggestion
    - name: pre-rafter
```

I don't see a point in explicitly saying that this is only of master